source/mainnet/net/resources/terms-and-conditions-bw.rst

```diff
@@ -0,0 +1,153 @@
+
+Terms and conditions for all services and functions
+===================================================
+
+Parties to the agreement
+------------------------
+
+When you register with |bw|, you enter into an agreement with:
+
+Concordium Software ApS, Amaliegade 4, 1256 København K, Denmark. CVR no.: 42334758. Our email address is contact@concordium.software.
+
+What you need in order to register with |bw|
+=================================================================
+
+The following is required to register with |bw|:
+
+- Have a computer with a supported browser version installed. You can see which browser versions can be used at `developer.concordium.software <https://developer.concordium.software>`_.
+- Access to download and install browser extensions in your computer’s browser.
+- Verify your identity with an identity provider. The identity provider will know your identity.
+
+You must be at least 18 years old to register.
+
+Registering as a |bw| user
+===============================================
+
+When you register in |bw|, you will generate private keys and you will receive a unique 24-word recovery phrase that must be stored in a safe place. The recovery phrase is the “master key” for all of your accounts – it is your private keys in mnemonic form. These words, when entered into another Concordium Wallet (not including the Cordordium Desktop Wallet) in the correct order, will recover all of the private keys you were storing on your original wallet. The recovery phrase is used as your personal backup of your private keys. If you lose the private keys and the recovery phrase, you will lose access to any funds in the account(s) permanently.
+
+Before you can use |bw|, you will need to create a digital identity certificate by following the instructions in the app. An identity provider is a third-party supplier and independent of Concordium Software. The |bw| stores data in encrypted form from the identity provider, which Concordium Software does not have access to.
+
+After you have created your digital identity certificate and your first account, you can create additional accounts with which you will be able to carry out transactions. Concordium Software may impose a limit to how many accounts you can create in a single |bw|.
+
+Activation code and personal password
+=====================================
+
+When you register in |bw|, you will need to create a personal password. If your computer supports it, you can subsequently activate your fingerprint or use facial recognition instead of the password. You should be aware that all fingerprints/facial recognitions linked to your computer can be used to log on to |bw| and use the functions. You are therefore obliged to ensure that you have only activated your own fingerprints and your own facial recognition if you want to use them instead of the password for access to |bw|. |bw| does not store your fingerprints or facial recognition.
```

Comment:
It's not possible to use fingerprint or facial recognition in the BW, so we probably have to change this section

Reply:
We just have to remember to change this part before we release.